backend/compact-connect/lambdas/python/data-events/tests/function/__init__.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import os
+
+import boto3
+from moto import mock_aws
+
+from .. import TstLambdas
+
+logger = logging.getLogger(__name__)
+logging.basicConfig()
+logger.setLevel(logging.DEBUG if os.environ.get('DEBUG', 'false') == 'true' else logging.INFO)
+
+
+@mock_aws
+class TstFunction(TstLambdas):
+    """Base class to set up Moto mocking and create mock AWS resources for functional testing"""
+
+    def setUp(self):  # noqa: N801 invalid-name
+        super().setUp()
+
+        self.build_resources()
+
+        from cc_common import config
+
+        config.config = config._Config()  # noqa: SLF001 protected-access
+        self.config = config.config
```

Comment:
isn't this already done by the super class?

Reply:
Good catch. Removing.